PD.Workademy.Todo/src/Web/PD.Workademy.Todo.Web/Controllers/CategoryController.cs

```diff
@@ -1,17 +1,122 @@
 ﻿using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.Mvc;
+using PD.Workademy.Todo.Web.ApiModels;
 
 namespace PD.Workademy.Todo.Web.Controllers
 {
-    [Route("api/[controller]")]
+    
     [ApiController]
```

Comment:
[ApiController] nije potreban posto vec postoji u base klasi... ovo vazi za sve kontrolere, tako da bih to sklonio